docs/gettingstarted.md

```diff
@@ -0,0 +1,66 @@
+# Getting Started
```

Comment:
I wrote this fairly quickly, so please let me know if you catch typos! Just expanded on the stuff you wrote in the README.md